portal-web/test/functional/com/liferay/portalweb/tests/MyTestFile/MyTestFile.testcase

```diff
@@ -1,29 +1,118 @@
 @component-name = "portal-web-forms-and-data-lists"
 definition {
-property testray.main.component.name = "Forms";
-setUp {
-TestCase.setUpPortalInstance();
-User.firstLoginPG();
-Navigator.openURL();
-}
-tearDown {
-User.logoutAndLoginPG(
-userLoginEmailAddress = "test@liferay.com",
-userLoginFullName = "Test Test");
-}
-
-@description = "Access Portal Global Menu and validate CONFIGURATION string at
-Control Panel"
-@priority = "5"
-test ValidateConfigurationStringAtControlPanel {
-task("accessing the option menu"){
-        MyTestFile.clickMenu();
-}
-task("click on control panel option"){
-        MyTestFile.clickControPanel();
-}
-task("check the configuration section is present"){
-        MyTestFile.assertConfiguration();
-}
-}
+
+	property testray.main.component.name = "Forms";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		Navigator.openURL();
+	}
+
+	tearDown {
+		User.logoutAndLoginPG(
+			userLoginEmailAddress = "test@liferay.com",
+			userLoginFullName = "Test Test");
+	}
+
+	@description = "validate entries in english and portuguese"
+	@priority = 5
+	test CheckIsPossibleSubmitFormWithTwoDifferentLanguages {
+		task ("add a form and an edited text field to full name, and back to another field") {
+			FormsAdmin.addForm();
+
+			Form.editName(formName = "New Forms");
+
+			Form.gotoAddField(
+				fieldPositionNumber = 0,
+				fieldType = "Text");
+
+			Form.editFieldLabelMetal(fieldValue = "Full Name");
+
+			Form.closeSidebar();
+		}
+
+		task ("add a numerical field and edit it for age") {
+			Form.gotoAddField(
+				fieldPositionNumber = 1,
+				fieldType = "Numeric");
+
+			Form.editFieldLabelMetal(fieldValue = "Age");
+		}
+
+		task ("adds a new translation for the form (portuguese) and edits full name to full name and age to age") {
```

Comment:
Use capital letters at the beginning of the sentence, use "edits full name and age" instead of "edits full name to full name and age to age"